packages/javascript/bh-shared-ui/src/components/UpdateUserDialog/UpdateUserDialog.test.tsx

```diff
@@ -61,12 +150,10 @@ const server = setupServer(
             })
         );
     }),
-    rest.get('/api/v2/saml', (req, res, ctx) => {
+    rest.get('/api/v2/sso-providers', (req, res, ctx) => {
```

Comment:
Same feedback here if you wanted to add the response type you can get some extra type safety